vicon2_driver/src/vicon2_driver.cpp

```diff
@@ -252,81 +238,133 @@ void ViconDriverNode::process_markers(const rclcpp::Time & frame_time, unsigned
       get_logger(), "IsMarkerDataEnabled? %s",
       client.IsMarkerDataEnabled().Enabled ? "true" : "false");
   }
+
   if (!unlabeled_marker_data_enabled_) {
     unlabeled_marker_data_enabled_ = true;
     client.EnableUnlabeledMarkerData();
 
-    RCLCPP_INFO(
+  RCLCPP_INFO(
       get_logger(), "IsUnlabeledMarkerDataEnabled? %s",
       client.IsUnlabeledMarkerDataEnabled().Enabled ? "true" : "false");
   }
+
   n_markers_ = 0;
   mocap_msgs::msg::Markers markers_msg;
   markers_msg.header.stamp = frame_time;
   markers_msg.frame_number = vicon_frame_num;
-  unsigned int UnlabeledMarkerCount = client.GetUnlabeledMarkerCount().MarkerCount;
 
-  RCLCPP_INFO(
-    get_logger(),
-    "# unlabeled markers: %d", UnlabeledMarkerCount);
-  n_markers_ += UnlabeledMarkerCount;
-  n_unlabeled_markers_ = UnlabeledMarkerCount;
-  for (unsigned int UnlabeledMarkerIndex = 0;
-    UnlabeledMarkerIndex < UnlabeledMarkerCount;
-    ++UnlabeledMarkerIndex)
+  // Count the number of subjects
+  unsigned int SubjectCount = client.GetSubjectCount().SubjectCount;
+  RCLCPP_DEBUG(get_logger(), "Subjects: (%u)", SubjectCount);
+  for( unsigned int SubjectIndex = 0 ; SubjectIndex < SubjectCount ; ++SubjectIndex )
```

Comment:
Does this line pass the linter? Did you do a `colcon test`

Reply:
I did a `ament_uncrustify` and changed the file.